### PR TITLE
#482 - added meta title field to cms pages

### PR DESCRIPTION
**Pull request by tzyganu**

### CODE REVIEW

**Comment by mazhalai** [comment]
@tzyganu  Please sync with develop. Builds have been fixed.

**Comment by hshar7** [comment]
Internal ticket: MAGETWO-50238

File: app/code/Magento/Cms/Setup/UpgradeSchema.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * Upgrade the Cms module DB scheme
+ */
+class UpgradeSchema implements UpgradeSchemaInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $setup->startSetup();
```
Comment (buskamuza) [inline app/code/Magento/Cms/Setup/UpgradeSchema.php]: start/end-Setup might not be necessary here. Better to not use it, if it's not needed. It disables FK validation.

**Comment by strajeru** [comment]
@mazhalai can you please restart the tests. some are failing with `The command "sudo apt-get update -qq" failed and exited with 100 during .`.

**Comment by mazhalai** [comment]
@strajeru Builds restarted.

**Comment by okorshenko** [comment]
@strajeru Thank you for contribution. Your Pull Request merged to develop branch.

**Pull request merged**